plugins/modules/vyos_command.py

```diff
@@ -166,8 +173,8 @@ def parse_commands(module, warnings):
 
 def main():
     spec = dict(
-        commands=dict(type="list", required=True),
-        wait_for=dict(type="list", aliases=["waitfor"]),
+        commands=dict(type="list", required=True, elements="str"),
```

Comment:
As discussed, setting `elements: str` here would break tasks that have a command-prompt-answer dict under `commands`. We can change it to `elements: raw` instead.